store/tikv/gcworker/gc_worker.go

```diff
@@ -155,7 +153,8 @@ func (w *GCWorker) start(ctx context.Context, wg *sync.WaitGroup) {
 	logutil.Logger(ctx).Info("[gc worker] start",
 		zap.String("uuid", w.uuid))
 
-	w.session = createSession(w.store)
+	se := createSession(w.store)
+	defer se.Close()
```

Comment:
It seems the session is not used in this function

Reply:
addressed